File: problemas/iniciante/1985/1985.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        double[] precos = new double[] { 1.5, 2.5, 3.5, 4.5, 5.5 };

        int N = int.Parse(Console.ReadLine());
        double total = 0.0;
        for (int i = 0; i < N; ++i)
        {
            string[] entrada = Console.ReadLine().Trim().Split(' ');

            total += int.Parse(entrada[1]) * precos[int.Parse(entrada[0]) - 1001];
        }

        Console.WriteLine($"{total:0.00}");
    }
}

File: problemas/iniciante/1985/1985.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);

        double[] precos = { 1.5, 2.5, 3.5, 4.5, 5.5 };

        int N = Integer.parseInt(in.readLine());
        double total = 0.0;
        for (int i = 0; i < N; ++i) {
            String[] entrada = in.readLine().trim().split(" ");

            total += Integer.parseInt(entrada[1]) * precos[Integer.parseInt(entrada[0]) - 1001];
        }

        System.out.printf("%.2f\n", total);
    }
}